src/core/experimental/memory/memcpy.d

```diff
@@ -0,0 +1,401 @@
+/**
+ * Pure D replacement of the C Standard Library memcpy().
+ * There is an idiomatic-D interface, memcpy(), which is split into 3 overloads.
+ * One taking static types, one dynamic arrays and one static arrays.
+ * Also, there is available a C-like interface, the Dmemcpy() (which is named Dmemcpy
+ * for disambiguation with the C memcpy() a _similar_ interface) that
+ * is the classic (void*, void*, size_t) interface.
+ * N.B.: Both the memcpy() here and Dmemcpy() return nothing, contrary to the C Standard
+ * Library version.
+ * Source: $(DRUNTIMESRC core/experimental/memory/memcpy.d)
+ */
```

Comment:
DStyle is without leading stars.

Reply:
I don't really get where to use the one and where the other.
For example, with leading stars: https://github.com/dlang/phobos/blob/master/std/algorithm/searching.d#L277
Without leading stars:
https://github.com/dlang/phobos/blob/master/std/algorithm/searching.d#L213

Reply:
If you're not sure, you can check the D Style document[1], which should always be followed for new code: "Documentation comments should not have leading stars on each line."

A lot of the code in druntime (and sadly also some in Phobos) is old and doesn't follow the style very closely or at all.

[1] https://dlang.org/dstyle.html

Reply:
Thanks, I'm trying to follow but things escape here and there.

Reply:
Yes there's lots of old code here and in Phobos that was never changed, but at least new code should follow the guidelines.

Reply:
Yes ok